src/Provision.js

```diff
@@ -347,11 +347,29 @@ const handleTenantFaucet = async ({tenantId, asUrl, t, debug}) => {
     amount = undefined;
   }
 
-  let res = await elvTenant.TenantCreateFaucetAndFund({
-    asUrl,
-    tenantId,
-    amount,
-  });
+  let maxAttempts = 3;
+  let backoffTime = 2 * 60 * 1000; // 2 minute
```

Comment:
don't wait 2 minutes between polls; start with a marginal backoff, say, 15 secs, and exponentially increase it until you hit a max of 2mins, then fail

15, 30, 60, 120, fail